content/advent-2016/contributing-to-the-go-project.md

```diff
@@ -0,0 +1,218 @@
++++
+author = ["Matt Layher"]
+date = "2016-12-08T08:00:00+00:00"
+title = "Contributing to the Go project"
+series = ["Advent 2016"]
++++
+
+For both rookie and veteran Gophers, contributing to the Go project can
+seem overwhelming.  The official
+[Contribution Guidelines](https://golang.org/doc/contribute.html) document
+is rather lengthy; enough so that an
+[issue has been filed](https://github.com/golang/go/issues/17802) to make
```

Comment:
Maybe just link "rather lengthy" to the issue directly without pointing it out specifically?